client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -361,6 +371,24 @@ export const useCommandsArrayWpcom = ( {
 			},
 			icon: cacheIcon,
 		},
+		{
+			name: 'visitSite',
+			label: __( 'Visit site' ),
```

Comment:
Can we add `...` to match what we have in a global page?

Also:
[ Proposal]: When you visit the site, it would be nice to have access to the command palette in there as well to easily transition to another site, or back to another screen, instead of using the mouse to go back. Is there a way to add that?

Reply:
>Can we add ... to match what we have in a global page?

The commands array include the raw label, and the `...` is added by the command palette component if it recognizes the user needs to do additional stuff to execute the command (e.g., pick a site.)

Reply:
> you visit the site, it would be nice to have access to the command palette in there as well to easily transition to another site

I agree with this suggestion, I think it makes a lot of sense to have access to the command palette on the front end to have an easy way to navigate back to the editing area. One possibility is to open the site in a new tav so that at least the editor area remains accessible in the other tab and the user is spared from awkward navigation experience